book/book.toml

```diff
@@ -0,0 +1,28 @@
+[book]
+authors = ["Josh Rotenberg"]
+language = "en"
+multilingual = false
+src = "src"
+title = "Clippy Documentation"
+
+[rust]
+edition = "2018"
+
+[output.html]
+edit-url-template = "https://github.com/rust-lang/rust-clippy/edit/master/guide/{path}"
+git-repository-url = "https://github.com/rust-lang/rust-clippy/tree/master/guide"
+mathjax-support = true
```

Comment:
Do we need this for the Clippy documentation?

Reply:
Only if we want the GitHub icon and a link to edit the docs (for example, on the mdbook guide it links to this: https://github.com/rust-lang/mdBook/tree/master/guide). We can just leave it out, too.

Reply:
It won't hurt to leave them in I assume. So nothing to do here.

Reply:
I changed it to `book` from `guide` so when we do actually deploy it it should point to the right place.